src/scripts/main.js

```diff
@@ -1 +1,61 @@
 'use strict';
+
+const bodyElement = document.querySelector('body');
+
+function successMessage(message) {
+  bodyElement.insertAdjacentHTML('afterbegin', `
+    <div class="success" data-qa="notification">
+      ${message}
+    <div>
+    `);
+}
+
+function errorMessage(message) {
+  bodyElement.insertAdjacentHTML('afterbegin', `
+    <div class="warning" data-qa="notification">
+      ${message}
+    <div>
+    `);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  bodyElement.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(Error('First promise was rejected'));
+  }, 3000);
+});
+
+const secondPromise = new Promise(resolve => {
+  bodyElement.addEventListener('mousedown', (evt) => {
+    if (evt.button === 0 || evt.button === 2) {
+      resolve('Second promise was resolved');
+    }
+  });
+});
+
+const thirdPromise = new Promise((resolve, reject) => {
+  bodyElement.addEventListener('click', () => {
```

Comment:
1) Change the resolved message to 'Third promise was resolved' for the thirdPromise.
2) Apply this condition
The thirdPromise should be resolved with a message Third promise was resolved only after both left and right clicks happened